lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -1714,6 +1714,8 @@ sub update_amazon_asin {
             { id => $release->id },
         );
     }
+
+    $self->_delete_from_cache($release_id);
```

Comment:
Shouldn't this technically be its own commit? :p

Reply:
Good point, moved it to a separate commit.